publications-2018.bib

```diff
@@ -557,7 +557,7 @@ @InProceedings{Deolmi2018
 
 @InProceedings{Chandrashekar2018,
   author    = {Chandrashekar, Praveen and Gallego-Valencia, Juan Pablo and Klingenberg, Christian},
-  title     = {{A Runge--Kutta Discontinuous Galerkin Scheme for the Ideal Magnetohydrodynamical Model}},
+  title     = {{A Runge-Kutta Discontinuous Galerkin Scheme for the Ideal Magnetohydrodynamical Model}},
```

Comment:
I don't think that this change is correct. I thought we'd prefer to use emdashes for compound names (see formatting on the [Wikipedia article](https://en.wikipedia.org/wiki/Runge%E2%80%93Kutta_methods)).

Reply:
There seems to be genuine variability. Here's a long description: https://en.wikipedia.org/wiki/Dash#En_dash

But we are mostly consistent: The majority of "Navier-Stokes" in our list of references just use a hyphen:
```
> grep avier *bib
publications-2005.bib:   title   = {A locally conservative LDG method for the incompressible Navier-Stokes equations},
publications-2005.bib:   title   = {Galerkin least-squares stabilization operators for the Navier-Stokes equations, A unified approach},
publications-2006.bib:   title   = {Adaptive discontinuous Galerkin methods with shock-capturing for the compressible Navier-Stokes equations},
publications-2006.bib:   title   = {Symmetric Interior Penalty DG Methods for the Compressible Navier-Stokes Equations I: Method Formulation},
publications-2006.bib:   title   = {Symmetric Interior Penalty DG Methods for the Compressible Navier-Stokes Equations II: Goal-Oriented A Posteriori Error Estimation},
publications-2007.bib:   title   = {A note on discontinuous Galerkin divergence-free solutions of the Navier-Stokes equations},
publications-2008.bib:   title   = {An optimal order interior penalty discontinuous Galerkin discretization of the compressible Navier-Stokes equations},
publications-2008.bib:   title   = {Energy norm a-posteriori error estimation for divergence-free discontinuous Galerkin approximations of the Navier-Stokes equations},
publications-2009.bib:   title   = {An equal-order DG method for the incompressible Navier-Stokes equations},
publications-2009.bib:   title   = {Numerical Methods for the Navier-Stokes Equations applied to Turbulent Flow and to Multi-Phase Flow},
publications-2009.bib:   title   = {Grad-div stabilization and subgrid pressure models for the incompressible Navier--Stokes equations},
publications-2009.bib:   title   = {A multilevel discontinuous Galerkin method for the compressible Navier-Stokes equations},
publications-2010.bib:   title   = {Stromfunktionsformulierung der Navier-Stokes-Gleichungen und Methoden der Druckrekonstruktion},
publications-2011.bib:   title   = {A Projection-Based Variational Multiscale Method for the Incompressible Navier--Stokes/Fourier Model},
publications-2013.bib:   title   = {Numerical flux functions for Reynolds-averaged Navier-Stokes and k$\omega$ turbulence model computations with a line-preconditioned p-multigrid discontinuous Galerkin solver},
publications-2013.bib:   title   = {Parameter Estimation for the Coupled Biot-Lame-Navier Problem in Subsurface Modeling},
publications-2014.bib:   title   = {The micropolar Navier-Stokes equations: A priori error analysis},
publications-2014.bib:   title   = {Discontinuous Galerkin discretization of the Reynolds-averaged Navier-Stokes equations with the shear-stress transport model},
publications-2015.bib:   title   = {Parallel multiphase Navier-Stokes solver},
publications-2015.bib:   title   = {Local projection FEM stabilization for the time-dependent incompressible Navier-Stokes problems},
publications-2015.bib:   title   = {A Discontinuous Galerkin Approach for the Unsteady Incompressible Navier--Stokes Equations},
publications-2015.bib:   title   = {A Posteriori Modeling Error Estimates for the Assumption of Perfect Incompressibility in the Navier--Stokes Equation.},
publications-2016.bib:   title   = {Stability analysis of pressure correction schemes for the Navier-Stokes equations with traction boundary conditions},
publications-2016.bib:  title     = {Solution of incompressible viscous time-dependent Navier-Stokes equations using projection method},
publications-2016.bib:  url       = {https://www.researchgate.net/profile/Muhammad_Mohebujjaman/project/Solution-of-incompressible-viscous-time-dependent-Navier-Stokes-equations-using-projection-method/attachment/57ad503e08aef1b475aef153/AS:394116325232643@1470976062899/download/project+_write.pdf?context=ProjectUpdatesLog},
publications-2016.bib:  title     = {{Parallel block preconditioning of the incompressible Navier-Stokes equations with weakly imposed boundary conditions}},
publications-2016.bib:  title     = {{The Time-Dependent Navier--Stokes Equations: Laminar Flows}},
publications-2017.bib:   title   = {On conservation laws of Navier-Stokes Galerkin discretizations},
publications-2017.bib:   title   = {On the stability of projection methods for the incompressible Navier-Stokes equations based on high-order discontinuous Galerkin discretizations},
publications-2017.bib:   title   = {Numerical analysis and comparison of high performance incompressible Navier Stokes equations solvers based on the discontinuous Galerkin Method},
publications-2018.bib:   title   = {A matrix-free high-order discontinuous Galerkin compressible Navier-Stokes solver: A performance comparison of compressible and incompressible formulations for turbulent incompressible flows},
```

Reply:
> publications-2015.bib:   title   = {A Posteriori Modeling Error Estimates for the Assumption of Perfect Incompressibility in the **Navier--Stokes** Equation.},

I remembered changing some of these, but clearly I didn't do it consistently. Ok, maybe when (if?) we go through and correct all of the entries with their DOI-scraped data this will change anyway. I'm happy if we leave this as it is.